tests/generation/test_streamers.py

```diff
@@ -102,9 +104,12 @@ def test_text_streamer_decode_kwargs(self):
         model.config.eos_token_id = -1
 
         input_ids = torch.ones((1, 5), device=torch_device).long() * model.config.bos_token_id
-        with CaptureStdout() as cs:
-            streamer = TextStreamer(tokenizer, skip_special_tokens=True)
-            model.generate(input_ids, max_new_tokens=1, do_sample=False, streamer=streamer)
+
+        root = _get_library_root_logger()
+        with patch.object(root, "propagate", False):
```

Comment:
It turns out that live log will interfere with `CaptureStdout` and cause this test failed.

Simply patch the root logger's `propagate` within `with` here.